apps/researcher/src/app/[locale]/research-guide/filterGuides.test.ts

```diff
@@ -0,0 +1,197 @@
+import {ResearchGuide} from '@colonial-collections/api/src/research-guides/definitions';
```

Comment:
I had forgotten to export the definitions 🥲 and just updated my PR. Would this work?

`import {ResearchGuide} from '@colonial-collections/api';`

(Also applies to the rest of this PR)